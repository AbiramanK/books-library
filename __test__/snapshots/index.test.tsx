import TestRenderer from "react-test-renderer";

import Head from "@/app/head";

describe("Snapshots", () => {
  it("App <Head /> should render title and description", () => {
    const tree = TestRenderer.create(<Head />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
