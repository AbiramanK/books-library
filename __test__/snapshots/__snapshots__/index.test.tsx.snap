// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Layout => App => <Head /> should render title and description 1`] = `
[
  <title>
    Books Library
  </title>,
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />,
  <meta
    content="Books Library is a webapp that lists the books"
    name="description"
  />,
  <link
    href="/favicon.ico"
    rel="icon"
  />,
]
`;
